src/zcash/address/zip32.h

```diff
@@ -165,6 +165,8 @@ uint256 ovkForShieldingFromTaddr(HDSeed& seed);
 
 namespace libzcash {
 
+typedef uint32_t AccountId;
```

Comment:
Account ids should be in the range [0, 2<sup>31</sup>). Is there any way to ensure this in C++?

Reply:
One could certainly create a wrapper class and guarantee safe construction that way; this typedef is of course just documentation (of a sort that we regularly use in numerous places in the codebase.) I'm not sure that it's worth the extra boilerplate at all the use sites for now.